pkgs/os-specific/linux/beegfs/kernel-module.nix

```diff
@@ -41,5 +41,6 @@ in stdenv.mkDerivation {
     platforms = [ "i686-linux" "x86_64-linux" ];
     license = licenses.gpl2;
     maintainers = with maintainers; [ markuskowa ];
+    broken = stdenv.lib.versionAtLeast kernel.version "4.18";
```

Comment:
cc @markuskowa

Reply:
Thanks for the heads up. I'll keep an eye on the issue.